### ISSUE TITLE
PermissionError: [WinError 5]

### ISSUE THREAD

**Issue by Gaia-X-Alter** [JacquesLucke/blender_vscode#132]
https://github.com/JacquesLucke/blender_vscode/blob/1449ff34988784242144251cc20a771412797f2d/pythonFiles/include/blender_vscode/load_addons.py#L42

_os.remove(path, *, dir_fd=None)
Remove (delete) the file path. If path is a directory, an [IsADirectoryError](https://docs.python.org/3/library/exceptions.html#IsADirectoryError) is raised. Use [rmdir()](https://docs.python.org/3/library/os.html#os.rmdir) to remove directories. If the file does not exist, a [FileNotFoundError](https://docs.python.org/3/library/exceptions.html#FileNotFoundError) is raised._

--https://docs.python.org/3/library/os.html

you **can not**  remove a pugin folder via **os.remove** when **the old plugin exists**

a simple problem left unsolved for 4 years?->#17
=_=#

**Comment by theoryshaw** [comment]
This worked for me!  Possible to put in a PR with this fix?

**Comment by theoryshaw** [comment]
related: https://community.osarch.org/discussion/comment/17160/#Comment_17160

**Comment by zebus3d** [comment]
This work for me:
```
if os.path.exists(link_path):  
        # os.remove(link_path)
        shutil.rmtree(link_path)
```

**Comment by Mateusz-Grzelinski** [comment]
indeed. If also leads to data loss in various cases. https://github.com/JacquesLucke/blender_vscode/blob/master/EXTENSION-SUPPORT.md
